pkgs/top-level/all-packages.nix

```diff
@@ -1024,6 +1024,10 @@ with pkgs;
 
   useOldCXXAbi = makeSetupHook { } ../build-support/setup-hooks/use-old-cxx-abi.sh;
 
+  ical2org = callPackage ../tools/misc/ical2org {};
```

Comment:
I guess this has been dropped since & that's why it shows up here?
needs removing

Reply:
Rebase mistake, sorry.

Reply:
Is the file tools/misc/ical2org/default.nix missing in this PR?

Reply:
I am not sure what you're asking, this PR is about `iceportal`. The above line was a rebase mistake by me and is removed.